ogip_spectra/io_ogip.py

```diff
@@ -212,6 +212,12 @@ def extract_spectrum(pha_table):
             acceptance = pha_meta["BACKSCAL"]
         spectrum_data["acceptance"] = acceptance
 
+        if "EXPOSURE" in pha_table.columns:
```

Comment:
Why would the exposure  be stored in the PHA table?
https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/node7.html